app/main.py

```diff
@@ -4,4 +1,24 @@
+import os
+import requests
+from requests import Response
+
+
+def get_weather(api_key: str, city: str = "Paris") -> Response:
+    base_url = "http://api.weatherapi.com/v1/current.json"
+    params = {"key": api_key, "q": city}
+    response = requests.get(base_url, params=params)
+    if response.status_code == 200:
+        data = response.json()
+        print(f"City: {data['location']['name']}")
+        print(f"Temperature: {data['current']['temp_c']}°C")
+        print(f"Condition: {data['current']['condition']['text']}")
```

Comment:
Consider returning the data instead of printing it directly within the function. This would make the function more versatile and easier to test.